bedboss/refgenome_validator/refgenomevalidator.py

```diff
@@ -0,0 +1,103 @@
+import pandas as pd
+
+
+class Validator:
+    """
+    Initialize validator class
+
+    1. filter out incompatible genomes
+    2. for remaining, determine tiers of compatibility
+
+
+    """
+
+    def __init__(self, bedfiles, genome_models):
+        # TODO ensure these are lists
+
+        self.bedfiles = bedfiles
+        self.genome_models = genome_models
+
+        self.compatibility_matrix = pd.DataFrame(
+            index=self.bedfiles, columns=self.genome_models
+        )  # non mutually exclusive  validation, i.e.
+
+        self.predicted_parent = self.predict_parent()
+
+    def predict_parent(self):
```

Comment:
we probably can't predict parent.

We have 2 use modes:
1. give us a species, we give you predictions among all refs of that species
2. you don't know the species; we give you prediction among everything we have.